<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mp3ConfigPage</class>
 <widget class="QWidget" name="mp3ConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MP3 encoding configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="mp3PresetLabel">
       <property name="text">
        <string>Preset:</string>
       </property>
       <property name="buddy">
        <cstring>mp3PresetCbx</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mp3PresetCbx">
       <property name="toolTip">
        <string extracomment="ererere">&lt;dt&gt;VBR medium&lt;/dt&gt;
&lt;dd&gt;By using a medium Variable BitRate, this preset should provide near transparency to most people and most music.&lt;/dd&gt;

&lt;dt&gt;VBR standard, VBR standard fast&lt;/dt&gt;
&lt;dd&gt;By using a standard Variable BitRate, this preset should generally be transparent to most people on most music and is already quite high in quality.&lt;/dd&gt;

&lt;dt&gt;VBR extreme, VBR extreme fast&lt;/dt&gt;
&lt;dd&gt;By using the highest possible Variable BitRate, this preset provides slightly higher quality than the standard mode if you have extremely good hearing or high-end audio equipment.&lt;/dd&gt;

&lt;dt&gt;VBR quality&lt;/dt&gt;
&lt;dd&gt;This Variable BitRate option lets you specify the output quality.&lt;/dd&gt;

&lt;dt&gt;fast option&lt;/dt&gt;
&lt;dd&gt;Enables the new fast VBR for a particular profile. This is recommended.&lt;/dd&gt;

&lt;dt&gt;CBR insane&lt;/dt&gt;
&lt;dd&gt;If you must have the absolute highest quality with no regard to file size, you'll achieve it by using this Constant BitRate.&lt;/dd&gt;

&lt;dt&gt;CBR kbps&lt;/dt&gt;
&lt;dd&gt;Using this Constant BitRate preset will usually give you good quality at a specified bitrate.&lt;/dd&gt;

&lt;dt&gt;ABR kbps&lt;/dt&gt;
&lt;dd&gt;Using this Average BitRate preset will usually give you higher quality than the Constant BitRate option for a specified bitrate.&lt;/dd&gt;
</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="mp3BitrateBox">
     <property name="title">
      <string>Use bitrate</string>
     </property>
     <layout class="QGridLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="mp3BitrateLabel">
        <property name="text">
         <string>Bitrate:</string>
        </property>
        <property name="buddy">
         <cstring>mp3BitrateCbx</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mp3BitrateCbx">
        <property name="toolTip">
         <string>Sets target bitrate (in kb/s).</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mp3QualityBox">
     <property name="title">
      <string>Use quality</string>
     </property>
     <layout class="QGridLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="mp3QualityLabel">
        <property name="text">
         <string>Quality:</string>
        </property>
        <property name="buddy">
         <cstring>mp3QualitySlider</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_28">
        <item>
         <widget class="QSlider" name="mp3QualitySlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mp3QualitySpin">
          <property name="maximum">
           <number>9</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="oggGainGroup_2">
     <property name="title">
      <string>ReplayGain</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="formLayout_8">
        <item row="0" column="0">
         <widget class="QLabel" name="mp3GainLabel">
          <property name="text">
           <string>Calculate gain:</string>
          </property>
          <property name="buddy">
           <cstring>mp3GainCbx</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="mp3GainCbx"/>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>136</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp3PresetCbx</tabstop>
  <tabstop>mp3BitrateCbx</tabstop>
  <tabstop>mp3QualitySlider</tabstop>
  <tabstop>mp3QualitySpin</tabstop>
  <tabstop>mp3GainCbx</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mp3QualitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mp3QualitySpin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mp3QualitySpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mp3QualitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>506</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
